zerver/lib/timestamp.py

```diff
@@ -40,3 +41,7 @@ def timestamp_to_datetime(timestamp: float) -> datetime.datetime:
 def datetime_to_timestamp(dt: datetime.datetime) -> int:
     verify_UTC(dt)
     return calendar.timegm(dt.timetuple())
+
+def datetime_to_precise_timestamp(dt: datetime.datetime) -> float:
+    verify_UTC(dt)
+    return time.mktime(dt.timetuple()) + dt.microsecond/1000000
```

Comment:
Isn’t this just `dt.timestamp()`?

Reply:
It is, thanks for pointing it out. Seems like I went about this in a roundabout way. I'll fix it.